pkg/plugin/linuxutil/ethtool_stats_linux_test.go

```diff
@@ -69,32 +74,71 @@ func TestReadInterfaceStats(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name: "test unsported interface",
+			opts: &EthtoolOpts{
+				errOrDropKeysOnly: false,
+				addZeroVal:        false,
+			},
+			statsReturn: nil,
+			statErr:     errInterfaceNotSupported,
```

Comment:
IMO, just define the error in-place and disable the linter. It makes it harder to read the test otherwise when the error is defined out-of-scope (but definitely follow the linter's advice outside of the test).